site/docs/components/grid-layout/examples.mdx

```diff
@@ -31,59 +31,57 @@ Content in the Grid Layout is displayed inside Grid Item components. This helps
 
 </LivePreview>
 
 <LivePreview componentName="grid-layout" exampleName="ResponsiveLayout">
 
 ### Responsive layout
 
-The Grid Layout’s responsive properties allow you to customize specific properties of the grid (such as the number of columns and rows) per breakpoint.
+The grid layout’s responsive properties allow you to customize specific properties of the grid (such as the number of columns and rows) per breakpoint.
 
-You can use the default [Salt breakpoints](/salt/foundations/responsiveness#breakpoints) or pass in your own using [SaltProvider](/salt/components/salt-provider).
+You can use the default [Salt breakpoints](/salt/foundations/responsiveness#breakpoints) or pass in your own using [`SaltProvider`](/salt/components/salt-provider).
 
-#### What are responsive properties?
-
-A responsive property is a property that takes either multiple values (one value per breakpoint), or a single value that would take effect across all different screen sizes. For example, a layout that will responsively change from 12 columns to two and then to one as the space decreases:
+A responsive property is a property that takes either multiple values (one value per breakpoint) or a single value that would take effect across all different screen sizes. An example is a layout that will responsively change from 12 columns to two and then to one as the space decreases:
 
 ```js
 columns={{ xs: 1, sm: 2, lg: 12, md: 12, xl: 12 }}
 ```
 
-#### Guidance
+#### Best practices
 
-- Use Grid Layout’s responsive properties when the flow of the grid is dependent on the viewport size, such as when a user typically resizes their application or webpage and content needs to remain visible.
+- Use `GridLayout`’s responsive properties when the flow of the grid is dependent on the viewport size, such as when a user typically resizes their application or webpage and content needs to remain visible.
 
-- Define specific layout behaviours per breakpoint to optimize the content display across different device sizes.
+- Define specific layout behaviors per breakpoint to optimize the content display across different device sizes.
 
 </LivePreview>
 
 <LivePreview componentName="grid-layout" exampleName="SpanningColumnsAndRows">
 
 ### Spanning columns and rows
 
-We strongly recommend that you display content in Grid Layout inside Grid Items. Using a Grid Item you can control the size and position of content in the grid. By default, a Grid Item will span one column and one row. Use the `colSpan` and `rowSpan` props to increase the size of individual items to span multiple rows and columns.
+We strongly recommend that you display content in `GridLayout` inside grid items. Using a grid item, you can control the size and position of content in the grid. By default, a grid item will span one column and one row. Use the `colSpan` and `rowSpan` props to increase the size of individual items to span multiple rows and columns.
```

Comment:
```suggestion
Although it is possible to use `GridLayout` without using `GridItem`, we strongly recommend that you wrap items with the `GridItem` component. Using a grid item, you can control the size and position of content in the grid. By default, a grid item will span one column and one row. Use the `colSpan` and `rowSpan` props to increase the size of individual items to span multiple rows and columns.
```
I think it would be something like this